client/src/components/PlayersLastPlayed/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Trophy } from '@styled-icons/bootstrap';
+import { PrimaryIcon } from '../IconStyles';
+
+const IconWrapper = styled.div`
+  ${PrimaryIcon};
+`;
+const PlayerWrapper = styled.div`
+  display: grid;
+  grid-template-columns: 34px 1fr auto;
+`;
+
+const Score = styled.div`
+  justify-self: center;
+`;
+
+const Player = ({ name, score, win, ...props }) => {
```

Comment:
You don't need to spread over props on every component, usually this is done if you are going to pass props down to another component
```suggestion
const Player = ({ name, score, win }) => {
```

Reply:
good to know